code/client/makecatalogs

```diff
@@ -150,6 +199,29 @@ def makecatalogs(repopath, options):
                 if key.startswith('_'):
                     del(pkginfo[key])
 
+
+            if iconhashing:
+
+                name = pkginfo.get('name')
+
+                if pkginfo.get('icon_name'):
+                    try:
+                        iconhash = icons.get(icon_name)
+                    except:
+                        del(pkginfo['icon_name'])
```

Comment:
Not so sure about deleting icon_name here, now admins don't get feedback when they use an icon_name that is pointing to a non-existent icon.